src/libstd/error.rs

```diff
@@ -197,9 +197,7 @@ pub trait Error: Debug + Display {
 
     /// Get the `TypeId` of `self`
     #[doc(hidden)]
```

Comment:
We should remove the doc(hidden), right?

Reply:
Hmm that’s a good point. Why was this hidden in the first place? It has been since the method was first added in https://github.com/rust-lang/rust/pull/24793. @aturon, do you remember?